py4DSTEM/io/nonnative/read_dm.py

```diff
@@ -75,19 +107,16 @@ def get_metadata_from_dmFile(fp):
     with dm.fileDM(fp, on_memory=False) as dmFile:
         pixelSizes = dmFile.scale
         pixelUnits = dmFile.scaleUnit
-        assert(pixelSizes[0]==pixelSizes[1]), "Rx and Ry pixel sizes don't match"
-        assert(pixelSizes[2]==pixelSizes[3]), "Qx and Qy pixel sizes don't match"
-        assert(pixelUnits[0]==pixelUnits[1]), "Rx and Ry pixel units don't match"
-        assert(pixelUnits[2]==pixelUnits[3]), "Qx and Qy pixel units don't match"
+        assert pixelSizes[0] == pixelSizes[1], "Rx and Ry pixel sizes don't match"
+        assert pixelSizes[2] == pixelSizes[3], "Qx and Qy pixel sizes don't match"
+        assert pixelUnits[0] == pixelUnits[1], "Rx and Ry pixel units don't match"
+        assert pixelUnits[2] == pixelUnits[3], "Qx and Qy pixel units don't match"
```

Comment:
Changed just for funsies?

Reply:
This is also an auto-format from running black